docs/try-it-out/longer-introduction.md

```diff
@@ -44,22 +44,22 @@ For this tutorial, use the [Schedule trigger](/integrations/builtin/core-nodes/n
 7. Close the node details view to return to the canvas.
 
 
-## Step four: Add the NASA node and set up credentials
+## Step three: Add the NASA node and set up credentials
 
-The [NASA node](/integrations/builtin/app-nodes/n8n-nodes-base.nasa.md) allows you to interact with NASA's [public APIs](https://api.nasa.gov/){:target=_blank .external-link}. The API gives you data to work with in this tutorial.
+The [NASA node](/integrations/builtin/app-nodes/n8n-nodes-base.nasa.md) interacts with NASA's [public APIs](https://api.nasa.gov/){:target=_blank .external-link} to fetch useful data. We will use the real-time data from the API to find solar events.
 
 1. Select the **Add node** <span class="inline-image">![Add node icon](/_images/try-it-out/add-node-small.png){.off-glb}</span> connector on the Schedule Trigger node.
-2. Search for **NASA**. n8n shows a list of nodes that match the search.
-3. Select **NASA** to view a list of operations.
-4. Search for and select **Get a DONKI solar flare**. This operation returns a report about recent solar flares. When you select the operation, n8n adds the node to the canvas and opens it.
-5. To access the NASA APIs, you need to set up credentials:
+1. Search for **NASA**. n8n shows a list of nodes that match the search.
+1. Select **NASA** to view a list of operations.
+1. Search for and select **Get a DONKI solar flare**. This operation returns a report about recent solar flares. When you select the operation, n8n adds the node to the canvas and opens it.
+1. To access the NASA APIs, you need to set up credentials:
     1. Select the  **Credential for NASA API** dropdown.
```

Comment:
If we're updating to use `1.` for all of the numbers, we might want to change this list too.